code/game/machinery/computer/card.dm

```diff
@@ -442,4 +499,43 @@ var/time_last_changed_position = 0
 	circuit = /obj/item/weapon/circuitboard/card/centcom
 	req_access = list(access_cent_commander)
 	change_position_cooldown = -1
-	blacklisted = list()
\ No newline at end of file
+	blacklisted = list()
+
+
+/obj/machinery/computer/card/minor
+	name = "department management console"
+	target_dept = 1
```

Comment:
Please create & use defines for all of these different target_dept values. I would recommend something like `TARGET_DEPARTMENT_ALL` through `TARGET_DEPARTMENT_x`